modules/conductor/src/main/java/org/opencastproject/event/handler/SeriesUpdatedEventHandler.java

```diff
@@ -275,19 +275,10 @@ public void handleEvent(final SeriesItem seriesItem) {
         barrier.waitForJobs();
       }
     } catch (SearchException e) {
-      logger.warn("Unable to find mediapackages in search: ", e.getMessage());
-    } catch (UnauthorizedException e) {
-      logger.warn(e.getMessage());
-    } catch (MediaPackageException e) {
-      logger.warn(e.getMessage());
-    } catch (ServiceRegistryException e) {
-      logger.warn(e.getMessage());
-    } catch (NotFoundException e) {
-      logger.warn(e.getMessage());
-    } catch (IOException e) {
-      logger.warn(e.getMessage());
-    } catch (DistributionException e) {
-      logger.warn(e.getMessage());
+      logger.warn("Unable to find mediapackages for series {} in search: {}", seriesItem, e.getMessage());
+    } catch (UnauthorizedException | MediaPackageException | ServiceRegistryException
+             | NotFoundException | IOException | DistributionException e) {
+      logger.warn("Unable to update mediapackages for series {}: {}", seriesItem, e.getMessage());
```

Comment:
This is great to tighten up the catch. But, the exception message might be null, as mentioned, and then we won't know the class name of the original exception (because we can't go to the line of the code where the log was made and see the catch class type).

Since this is a WARN anyway, and should not be too frequent in the logs. How do you feel about including 3 more bits of info here: the username, mpId (if not null at this point), the exception class name being caught?

Reply:
I added username and exception class name. Hesitant to add the mp because the exceptions could be caused elsewhere in the try/catch where the mpid is not relevant. So to log an mpid accurately maybe needs additional try/catch blocks or some refactoring, a task I will be happy to leave to other maintainers :-)